MusicBandSimulator/MusicBandSimulator/IMusicInstrument.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MusicBandSimulator
+{
+    /// <summary>
+    /// Інтерфейс, який містить Назву інструмента та звук, з яким грає цей інструмент.
+    /// </summary>
+    internal interface IMusicInstrument
+    {
+        public string InstrumentName { get; set; }
```

Comment:
When the interface name is `IMusicInstrument` no need to add Instrument as a prefix of properties. 
`InstrumentName` -> `Name`

Reply:
done